src/pages/documentation/platform-assurance/tutorials/versions.md

```diff
@@ -0,0 +1,27 @@
+# Versions Assurance View
```

Comment:
Since this is a new file, we also need to include it in the summary to be accessible by the user in the docs site - see gatsby-config.js